demos/instr.h

```diff
@@ -45,6 +45,19 @@ void UnwindStackAndSlowlyReturnTo(const void *address);
 // (meltdown).
 extern char afterspeculation[];
 
+// Yields serializing instruction.
+// Must be inlined in order to avoid mispredicted Spectre v2 jumps over it.
```

Comment:
Technically, _technically_, we're calling those Spectre V1 jumps since they don't seek to poison the branch predictor.

Maybe we just avoid the whole mess and say "to avoid misprediction that skips the call"

Reply:
@chandlerc Hi Chandler, do you agree with the statement that it is also Spectre v1?

Reply:
Fixed.